zerver/tests/test_realm.py

```diff
@@ -632,21 +632,18 @@ def test_change_plan_type(self) -> None:
         self.assertEqual(realm.upload_quota_gb, None)
 
         do_change_plan_type(realm, Realm.STANDARD)
-        realm = get_realm('zulip')
```

Comment:
These calls aren't unnecessary; the point is to refetch the object from the database in case it wasn't saved properly.